src/utils.ts

```diff
@@ -215,21 +219,86 @@ export const setCursor = (canvas: HTMLCanvasElement | null, cursor: string) => {
   }
 };
 
+let eraserCanvasCache: any;
+let previewDataURL: string;
+export const setEraserCursor = (
+  canvas: HTMLCanvasElement | null,
+  theme: AppState["theme"],
+) => {
+  const cursorImageSizePx = 20;
+
+  const drawCanvas = () => {
+    eraserCanvasCache = document.createElement("canvas");
+    eraserCanvasCache.height = cursorImageSizePx;
+    eraserCanvasCache.width = cursorImageSizePx;
+    const context = eraserCanvasCache.getContext("2d")!;
+
+    context.beginPath();
+    context.arc(
+      eraserCanvasCache.width / 2,
+      eraserCanvasCache.height / 2,
+      5,
+      0,
+      2 * Math.PI,
+    );
+
+    context.lineWidth = 2;
+    context.fillStyle = oc.white;
+
+    if (theme === THEME.DARK) {
+      context.strokeStyle = oc.white;
+    } else {
+      context.strokeStyle = oc.black;
+    }
+    context.stroke();
+
+    context.beginPath();
+    context.arc(
+      eraserCanvasCache.width / 2,
+      eraserCanvasCache.height / 2,
+      7,
+      0,
+      2 * Math.PI,
+    );
+    context.strokeStyle = oc.white;
+    context.stroke();
+
+    previewDataURL = eraserCanvasCache.toDataURL(MIME_TYPES.svg) as DataURL;
+  };
+  if (!eraserCanvasCache) {
+    drawCanvas();
+  } else {
+    const context = eraserCanvasCache.getContext("2d")!;
+    const strokeStyle = context.strokeStyle;
+    if (theme === THEME.LIGHT && strokeStyle !== oc.black) {
+      drawCanvas();
+    } else if (theme === THEME.DARK && strokeStyle !== oc.white) {
+      drawCanvas();
+    }
```

Comment:
since we now have two outlines, this check now always returns `true` in light theme